views/confirm-overseas-entity-details.html

```diff
@@ -0,0 +1,58 @@
+{% extends "layout.html" %}
+
+{% set title = "Confirm Overseas Data" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    {% include "includes/list/errors.html" %} 
+    <h1 class="govuk-heading-xl">Confirm this is the correct company (NOT LIVE)</h1>
+
+  {% from "govuk/components/table/macro.njk" import govukTable %}
```

Comment:
This needs to be translated into an Overseas Entity. It's an example taken from another service. https://companieshouse.atlassian.net/browse/UAR-93